Projects/Chat/Sources/Views/MessageView.swift

```diff
@@ -104,6 +104,8 @@ struct MessageView: View {
                     LinkView(
                         vm: .init(url: url)
                     )
+                case let .action(action):
+                    ActionView(action: action)
```

Comment:
Can we add here                         .
`environment(\.colorScheme, .light)
`?